# avoid the use of system include and lib directories
# take the STAGING_INSTALL_PREFIX variable from environment as defined by spk Makefile
--- setup.py.orig	2020-06-30 08:55:24.000000000 +0000
+++ setup.py	2020-12-14 21:30:38.387665662 +0000
@@ -29,13 +29,14 @@
 
 NAME = "Pillow"
 PILLOW_VERSION = get_version()
-FREETYPE_ROOT = None
+lib_include = os.path.join(os.environ['STAGING_INSTALL_PREFIX'], "lib"), os.path.join(os.environ['STAGING_INSTALL_PREFIX'], "include")
+FREETYPE_ROOT = lib_include
 IMAGEQUANT_ROOT = None
 JPEG2K_ROOT = None
-JPEG_ROOT = None
+JPEG_ROOT = lib_include
 LCMS_ROOT = None
 TIFF_ROOT = None
-ZLIB_ROOT = None
+ZLIB_ROOT = lib_include
 
 
 if sys.platform == "win32" and sys.version_info >= (3, 9):
@@ -508,13 +509,8 @@
 
         # standard locations
         if not self.disable_platform_guessing:
-            _add_directory(library_dirs, "/usr/local/lib")
-            _add_directory(include_dirs, "/usr/local/include")
-
-            _add_directory(library_dirs, "/usr/lib")
-            _add_directory(include_dirs, "/usr/include")
-            # alpine, at least
-            _add_directory(library_dirs, "/lib")
+            _add_directory(library_dirs, os.environ['STAGING_INSTALL_PREFIX'] + "/lib")
+            _add_directory(include_dirs, os.environ['STAGING_INSTALL_PREFIX'] + "/include")
 
         if sys.platform == "win32":
             if PLATFORM_MINGW:
